packages/flutter_tools/lib/src/linux/build_linux.dart

```diff
@@ -19,15 +21,32 @@ Future<void> buildLinux(LinuxProject linuxProject, BuildInfo buildInfo) async {
     ..createSync(recursive: true)
     ..writeAsStringSync(Cache.flutterRoot);
 
+  final String cacheDirectory = artifacts.getEngineArtifactsPath(TargetPlatform.linux_x64);
+  String stampContents;
+  if (artifacts is LocalEngineArtifacts) {
+    // If we're working with a local engine, output the libflutter_linux.so timestamp
+    // into a file that can be used as a stamp for make.
+    stampContents = fs.statSync(fs.path.join(cacheDirectory, 'libflutter_linux.so')).modified.toIso8601String();
+  } else {
+    // Otherwise we can use the engine stamp.
+    stampContents = Cache.instance.getStampFileFor('linux-sdk').readAsStringSync();
+  }
+  linuxProject.editableHostAppDirectory.childFile('.generated_engine_stamp')
```

Comment:
Per comments on the other PR, I'm not sure it's useful to add this logic until flutter_tool is also doing the copy. At that point, it'll be easy to read the file and do logic on it (the way the current FDE copy script does).

Reply:
I've moved the copy the logic to the flutter tool, per my understanding of what the Makefile is doing...